trin-core/src/utils/db.rs

```diff
@@ -34,3 +44,15 @@ pub fn get_default_data_dir() -> String {
         None => panic!("Unable to find data directory"),
     }
 }
+
+pub fn cleanup_data_dir() {
+    let trin_config = TrinConfig::from_cli();
+    if trin_config.ephemeral {
```

Comment:
It seems like this current design assumes that we'll end up with duplicate data dirs inside `TRIN_NODE_ID_DATA_DIRS`, since every time we call `get_data_dir` the path gets added. Which is going to happen more than once for a single node id. Though, we also want to support removing multiple, different data dirs since currently the state network [creates a random one for triedb to use](https://github.com/ethereum/trin/blob/4e0a885b2c7705bee00cd891d81904369303e149/trin-state/src/network.rs#L38). However, this isn't what we want longterm since eventually triedb should use the same db as `PortalStorage`. 

I'm not a fan of failing silently here, but I do agree that it's maybe the best solution for right now, until we merge `TrieDb` with `PortalStorage`'s db. I'd suggest adding a `todo` comment here explaining that once we only use a single data dir, that we should update `TRIN_NODE_ID_DATA_DIRS` to just be a single string and to actually panic if there's an error with removing that dir.

Reply:
👍